zabbix_metrics_watcher.py

```diff
@@ -52,47 +51,53 @@ def is_finished(self):
     def duration(self):
         try:
             return int(self._data['status']['duration'])/1000000000
-        except:
+        except Exception as e:
+            logger.warn('Error duration: %s' % repr(e))
             return ""
 
     @property
     def upload_size_mb(self):
         try:
             tar_metadata = json.loads(self._data['metadata']['annotations']['tar_metadata'])
             return int(tar_metadata['size']) / (1024 * 1024)
-        except:
+        except Exception as e:
+            logger.warn('Error upload_size_mb: %s' % repr(e))
             return 0
 
     @property
     def durations(self):
         try:
             metadata = json.loads(self._data['metadata']['annotations']['plugins-metadata'])
             return metadata['durations']
-        except:
+        except Exception as e:
+            logger.warn('Error durations: %s' % repr(e))
             return {}
 
     @property
     def created_time(self):
         try:
             timestamp = self._data['metadata']['creationTimestamp']
             return dateutil.parser.parse(timestamp)
-        except:
+        except Exception as e:
+            logger.warn('Error created_time: %s' % repr(e))
             return None
 
     @property
     def started_time(self):
         try:
-            timestamp = self._data['metadata']['startTimestamp']
+            timestamp = self._data['status']['startTimestamp']
```

Comment:
IIRC startTimestamp is recorded in metadata after the build is complete (but not while the build is running), so it took some time to fix correctly